internal/controlplane/handlers_ruletype_test.go

```diff
@@ -1,228 +0,0 @@
-// Copyright 2023 Stacklok, Inc
```

Comment:
Deleted this file because:
1) The test cases have been moved to the unit tests for the rule service
2) I want to do some refactoring in my next PR to make it easier to write unit tests for the control plane
3) The remaining implementation of `CreateRuleType` is relatively trivial so we are not losing much coverage